File: backend/mini-xrm/src/main/java/hu/rrsoftware/minixrm/partner/dto/PartnerResponseDto.java
package hu.rrsoftware.minixrm.partner.dto;

import hu.rrsoftware.minixrm.enums.PartnerStatus;
import hu.rrsoftware.minixrm.enums.QualificationType;
import lombok.*;

import java.util.Set;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PartnerResponseDto {
    private Long id;
    private String name;
    private String taxNumber;
    private String headquarters;
    private PartnerStatus status;
    private Set<QualificationType> qualifications;
}

File: frontend/src/app/components/partner/dto/PartnerResponseDto.ts
import {PartnerStatus} from '../../enums/PartnerStatus';
import {QualificationType} from '../../enums/QualificationType';

export interface PartnerResponseDto {
  id: number;
  version: number;
  name: string;
  taxNumber: string;
  headquarters: string;
  status: PartnerStatus;
  qualifications: QualificationType[];
}
